examples/intro_data_profiler.ipynb

```diff
@@ -253,168 +196,208 @@
+   "outputs": [],
    "source": [
-    "One of the interesting features of the DataProfiler is the ability to update profiles from batches of data, which is then makes DataProfiler applicable for data streaming usage. In this section, the original dataset is separated into two batches with equal size. Each batch is then updated with DataProfiler sequentially.  \n",
+    "# run data profiler and get the report\n",
+    "my_dataframe = pd.DataFrame([[1, 2.0],[1, 2.2],[-1, 3]], columns=[\"col_int\", \"col_float\"])\n",
+    "profile = dp.Profiler(my_dataframe)\n",
+    "report  = profile.report(report_options={\"output_format\":\"compact\"})\n",
     "\n",
-    "After the update, we expect the resulted profiles give the same statistics as the profiles updated from the full dataset. We will verify that through some properties in `global_stats` of the profiles including `column_count`, `row_count`, `row_is_null_ratio`, `duplicate_row_count`. "
+    "# Print the report\n",
+    "print(json.dumps(report, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b16648ba",
+   "metadata": {},
+   "source": [
+    "## Profiler options"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b0cc8ad",
+   "metadata": {},
+   "source": [
+    "The Data Profiler can enable/disable statistics and modify features through profiler options. For example, if the users only want the statistics information, they may turn off the Data Labeler functionality. Below, let's remove the histogram and data labeler component while running Data Profiler."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "9f78f331",
+   "id": "bbac3a2c",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# read the input data and devide it into two equal halves\n",
-    "data = dp.Data(os.path.join(data_path, \"csv/aws_honeypot_marx_geo.csv\"))\n",
-    "df = data.data\n",
-    "df1 = df.iloc[:int(len(df)/2)]\n",
-    "df2 = df.iloc[int(len(df)/2):]\n",
-    "\n",
-    "# Update the profile with the first half\n",
-    "profile = dp.Profiler(df1)\n",
+    "profile_options = dp.ProfilerOptions()\n",
+    "profile_options.set({\"histogram_and_quantiles.is_enabled\": False,\n",
+    "                     \"data_labeler.is_enabled\": False,})\n",
     "\n",
-    "# Update the profile with the second half\n",
-    "profile.update_profile(df2)\n",
+    "profile = dp.Profiler(my_dataframe, profiler_options=profile_options)\n",
+    "report  = profile.report(report_options={\"output_format\":\"pretty\"})\n",
     "\n",
-    "# Update profile with the full dataset\n",
-    "profile_full = dp.Profiler(df)\n",
+    "# Print the report\n",
+    "print(json.dumps(report, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "590ca50b",
+   "metadata": {},
+   "source": [
+    "Besides toggling on and off features, other options like the data labeler sample size or histogram bin method can be directly set and validated as shown here:"
```

Comment:
imo, we should add a small comment explaining what's the sample size of the data labeler